src/clients/twitch.ts

```diff
@@ -0,0 +1,150 @@
+import { conjoinedMsg, twitchMsg } from '../messageObjects';
+import { authenticateTwitch } from '../oauth';
+import { Message } from 'discord.js';
+import bridge, { genericPromiseError } from './bridge';
+import configFile from '../../config.json';
```

Comment:
Instead of importing the config file separately in both `twitch.ts` and `discord.ts`, one thing we could try is using `export ... from` syntax within your `bridge.ts` file. This way we have a single reference that we don't need to change anywhere but the bridge itself

e.g:

```ts
export configFile from '../../config.json';
```

[mozilla doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export#re-exporting_aggregating)

Reply:
oh wait I need to see if that even works with json in the first place...

Reply:
The way I'm doing it is I'm just doing 
```ts 
import configFile from '../../config.json';

// the bridge stuff

export {
	configFile
};
```

Reply:
The format of `export configFile from '../../config.json'` does not work in Typescript; no clue why it don't but it don't!

Reply:
We are currently importing & exporting our config in the bridge now, although through the way above and not the syntax shown due to it not working. I'm gonna resolve this conversation.